src/Xamarin.Android.Build.Tasks/Tasks/MakeBundleNativeCodeExternal.cs

```diff
@@ -66,6 +70,20 @@ public override bool Execute ()
 			return !Log.HasLoggedErrors;
 		}
 
+		private static string ParseI18nFlags (string i18n)
```

Comment:
ohh, that's beautiful, thanks! if we need it (see other comments), I'll change it to something like that.